### DISCUSSION TITLE
Unable to find Eufy Security in HACS

### DISCUSSION THREAD

**Discussion by dobje** [fuatakgun/eufy_security#745] [Q&A]
Hi All,

I hope you can help me out maybe i missed something.

I have installed the Eufy Security Add-On (version 1.3) and it's up and running and also the RTSP Simple server Add-on ( version 0.17.6 )  but in step 3 from the installation ( i did already had hacs installed ) i cannot seem to find the Eufy Security inside HACS Integrations.

Hacs version 1.30.1 and HomeAssistant 2023.2.3 ( Supervisor 2012.01.1 OS 9.5 ) 

Did i miss a step ?

**Comment by gonzoweb-git** [comment]
I have exactly the same problem.
Eufy Security Add-On (version 1.3)
RTSP Simple server Add-on ( version 0.17.6 )
Hacs version 1.30.1 and HomeAssistant 2023.2.5

Is there a manual install?
I noticed there is no Eufy folder under config/custom_components

Something appears to be working because there are a lot of connect/disconnect messages (i have replaced the actual IP with xxx) with lots of different ports. Here is a typical example

2023-02-21 15:44:06.043  INFO  Connected to station T8030P2322381EB8 on host 192.168.xxx.xxx and port 26624 
2023-02-21 15:45:06.449  INFO  Disconnected from station T8030P2322381EB8

**Reply by KillaDreams** [reply]
No need to replace the IP as that is an internal address.

**Comment by KillaDreams** [comment]
Installing Eufy Security Integration

1- If you have not already installed, install HACS following this guide: https://hacs.xyz/docs/setup/download

2- When HACS is ready, search for Eufy Security inside HACS Integrations

3- Install Eufy Security integration, restart your Home Assistant instance.

4- Check Settings -> Devices & Services page (https://your-instance.duckdns.org/config/integrations), click on Add Integration and search for Eufy Security. If you do not see it, first validate that it is installed via HACS and you had restarted it, later try with another browser. Integrations list might be already cached in your browser.

**Comment by gonzoweb-git** [comment]
Hiya,

I’ve tried all that … many, many times.

Here are some screenshots to show what I mean.

There is obviously interaction with the add-on, as can be seen when changing the alarm settings, but there is no Dufy Security showing in the Integration to add it.

Very strange
Regards,

Gary

---------------------------------------
"Think about how stupid the average
person is, and then realise that half
of 'em are stupider than that."
